project/Dependencies.scala

```diff
@@ -57,7 +57,7 @@ object Dependencies {
 
   val logbackClassic: ModuleID =  "ch.qos.logback"              % "logback-classic" % "1.2.3"
   val scalaLogging: ModuleID =    "com.typesafe.scala-logging"  %% "scala-logging"  % scalaLoggingV
-  val swaggerUi: ModuleID =       "org.webjars"                 % "swagger-ui"      % "3.45.0"
+  val swaggerUi: ModuleID =       "org.webjars"                 % "swagger-ui"      % "3.46.0"
```

Comment:
Note swagger updates typically require a code change here: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/SwaggerRoutes.scala#L13

Reply:
Actually, @scala-steward pointed that out:

>Files still referring to the old version number
>The following files still refer to the old version number (3.45.0).
>You might want to review and update them manually.
>```
>http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/SwaggerRoutes.scala
>```

@fthomas it would be pretty I think if @scala-steward listed links to the files (and lines numbers too) rather than just their paths. 🤔

Reply:
Oh neat, I didn't even notice that! +1 on linking and making it less easy to miss 😀

Reply:
That's a great idea, @bl-ue! Feel free to open an issue for this idea in the [Scala Steward repo](https://github.com/scala-steward-org/scala-steward).

Reply:
@fthomas https://github.com/scala-steward-org/scala-steward/issues/2050 🎆